### DISCUSSION TITLE
Add .env file to gitignore

### DISCUSSION THREAD

**Discussion by anoushk1234** [vercel/next.js#40634] [Ideas]
Far too many times I've wanted to quickly build something and the .env file gets pushed to github, can we please add .env to gitignore?

I have seen prev discussions on this topic but it's not been take seriously, the cost of adding this not much and most people use .env and not .env.local

**Comment by elkhayate** [comment]
If your question is how you can add .enn file to gitignore file, is by simply adding .env to this file but make sure the name of the file should be '.gitignore' notice that you are missing a dot.

**Comment by emreunsal1** [comment]
As a solution, delete the .env file and recreate it. I assume that first creating the .env file and then adding the .gitignore rule is causing some issues. This way, the problem was resolved for me.

**Reply by emreunsal1** [reply]
Certainly, after deleting the .env file, don't forget to push the changes. :)
